containers/codespaces-linux/.devcontainer/library-scripts/sshd-debian.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Syntax: ./sshd-debian.sh [SSH Port (don't use 22)] [non-root user] [start sshd now flag] [new password for user]
+
+# Note: You can change your user's password with "sudo passwd $(whoami)" (or just "passwd" if running as root).
+
+SSHD_PORT=${1:-"2222"}
```

Comment:
We're working on freeing up port 22 on the host, so the default SSH port should work in Codespaces in a week or two, at which point we can probably move this back to the default since that will be simpler for folks to use with standard SSH config on the clients.

Reply:
Yeah, agreed.  We can PR that separately so this branch doesn't get too far out of date.